test/test_showinfo.py

```diff
@@ -1,15 +1,23 @@
 import os
+import pytest
 import re
-from cfbs.commands import info_command
+from cfbs.commands import info_command, clear_definition
 
-os.chdir(os.path.join(os.path.dirname(__file__),"sample"))
+@pytest.fixture(scope="function")
+def chdir(request):
+    os.chdir(os.path.join(os.path.dirname(__file__), 'sample'))
+    yield
+    os.chdir(request.config.invocation_dir)
```

Comment:
Should only be defined in one place.

Reply:
I didn't try this: https://docs.pytest.org/en/6.2.x/example/parametrize.html#indirect-parametrization

Will try now. :)

Reply:
worked. :tada: